h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -226,23 +225,20 @@ class SearchBarController extends Controller {
           // appended to the container element.
           event.preventDefault();
           this._input.value = '';
+          updateHiddenInput();
         }
       };
 
-      var handleEnterKey = (event) => {
```

Comment:
Note to self: We don't need this any more because the hidden input is always kept up to date.